src/components/molecules/BenefitFilter.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+import { FilteredBenefitsCounter } from "../atoms/FilteredBenefitsCounter";
+
+/**
+ * This component displays three navigation buttons which filters out the result of benefits
+ */
+export function BenefitFilter(props) {
+  const handleClick = () => props.filter();
+
+  return (
+    <div>
+      <p className="text-gray-700 leading-none font-bold">{props.text}</p>
+      <div className="w-full md:flex">
+        <span>
+          <ActionButton
+            text={props.eligible}
+            invert={props.isSelectedEligible === false}
+            onClick={handleClick}
+          >
+            <FilteredBenefitsCounter count={props.eligibleCount} />
+          </ActionButton>
+        </span>
+        <span className="md:ml-2">
+          <ActionButton
+            text={props.help}
+            invert={props.isSelectedHelp === false}
+            onClick={handleClick}
+          >
+            <FilteredBenefitsCounter count={props.helpCount} />
+          </ActionButton>
+        </span>
+        <span className="md:ml-2">
+          <ActionButton
+            text={props.others}
+            invert={props.isSelectedOthers === false}
+            onClick={handleClick}
+          >
+            <FilteredBenefitsCounter count={props.othersCount} />
+          </ActionButton>
+        </span>
+      </div>
+    </div>
+  );
+}
+
+BenefitFilter.propTypes = {
+  /**
+   * Text on top of the buttons
+   */
+  text: PropTypes.string,
+
+  /**
+   * Button text that displays Eligible benefits
+   */
+  eligible: PropTypes.string,
+
+  /**
+   * Displays the number of results of eligible benefits
+   */
+  eligibleCount: PropTypes.number,
+
+  /**
+   * Button text that displays Potential help
+   */
+  help: PropTypes.string,
```

Comment:
Sorry should have specified this before. Please add Text at the end of text props 
`helpText`
`otherText` 
ect ect